numpy/polynomial/tests/test_polynomial.py

```diff
@@ -102,6 +102,18 @@ def test_polydiv(self):
                 res = poly.polyadd(poly.polymul(quo, ci), rem)
                 assert_equal(res, tgt, err_msg=msg)
 
+    def test_polypow(self):
+        for i in range(5):
+            for j in range(5):
+                msg = "At i=%d, j=%d" % (i, j)
+                c = np.random.randint(0, 100, i + 1)
```

Comment:
I worry that on 32-bit platforms this could overflow - `100 ** 5` doesn't fit in a 32-bit integer.

Are you sure you need a random test here? None of the other tests are random

Reply:
I don't suppose they need to be, no. I saw some of the other tests in the package were and thought it might be a good idea, thanks for pointing out why it isn't.